basicrta/tests/test_combine_contacts.py

```diff
@@ -0,0 +1,230 @@
+"""
+Tests for combining contact timeseries from multiple repeat runs.
+"""
+
+import os
+import pytest
+import numpy as np
+import pickle
+import tempfile
+import shutil
+
+from basicrta.contacts import CombineContacts
+
+
+class MockAtomGroup:
+    """Mock atom group that can be pickled."""
+    def __init__(self, resids):
+        self.residues = MockResidues(resids)
+        
+
+class MockResidues:
+    """Mock residues that can be pickled."""
+    def __init__(self, resids):
+        self.resids = np.array(resids)
+
+
+@pytest.fixture
+def temp_dir():
```

Comment:
Instead of creating your temp_dir fixture, just use pytest's `tmp_path` fixture, see https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmp-path-fixture